services/web/server/src/simcore_service_webserver/projects/projects_handlers.py

```diff
@@ -69,9 +70,11 @@ async def open_project(request: web.Request) -> web.Response:
         )
 
         # user id opened project uuid
-        await projects_api.run_project_dynamic_services(
-            request, project, req_ctx.user_id, req_ctx.product_name
-        )
+        with contextlib.suppress(ProjectStartsTooManyDynamicNodes):
+            # NOTE: this allows for starting projects with less than a specific amount of dy-nodes
```

Comment:
It is not clear to me why this is the case.  Could you explain better why?

Reply:
attempt at doing this.